packages/schematics/src/utils.ts

```diff
@@ -396,3 +418,15 @@ export function determineTargetProjectName(
   }
   return null;
 }
+
+/**
+ * Checking if the target project has e2e setup
+ * Method will check if angular project architect has e2e configuration to determine if e2e setup
+ */
+export function determineTargetProjectHasE2E(
+  angularJSON: any,
+  projectName: string,
+): boolean {
+  const { architect } = angularJSON.projects[projectName];
```

Comment:
`architect` can also be called `targets`, we already have a utility function to normalize this within this file called `getTargetsConfigFromProject()` so you just need to use that here